src/components/Header/Header.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import { ReactComponent as Arrow } from '../../icons/back-arrow.svg'
+
+export default function Header({ title, onBack }) {
+  return (
+    <HeaderStyled>
+      <ArrowStyled onClick={onBack} />
+      <Headline>{title}</Headline>
+    </HeaderStyled>
+  )
+}
+Header.propTypes = {
+  title: PropTypes.string.isRequired,
+  onBack: PropTypes.func.isRequired,
+}
+const HeaderStyled = styled.section`
```

Comment:
You could consider user the `header` element instead of `section` as this is a header.